src/HomePage/components/BottomBar.js

```diff
@@ -102,82 +103,58 @@ class BottomBar extends Component {
           label="Canteen"
           classes={{
             selected: classes.selected,
-            label: classes.label
+            label: classes.label,
           }}
           onClick={() => {
-            history.push("/homepage/canteen");
+            history.push('/homepage/canteen');
           }}
           icon={<Home className={classes.icon} />}
         />
         <BottomNavigationAction
           label="Bookmark"
           classes={{
             selected: classes.selected,
-            label: classes.label
+            label: classes.label,
           }}
           onClick={() => {
-            history.push("/homepage/bookmark");
+            history.push('/homepage/bookmark');
           }}
-          icon={
-            bookmark.length > 0 ? (
```

Comment:
<img width="341" alt="screenshot 2018-11-29 at 5 22 33 pm" src="https://user-images.githubusercontent.com/12982599/49212314-b5ab2300-f3fc-11e8-80cf-5b11e8a8549b.png">
<img width="832" alt="screenshot 2018-11-29 at 5 31 20 pm" src="https://user-images.githubusercontent.com/12982599/49212316-b6dc5000-f3fc-11e8-824c-13fe34e1aed2.png">

For displaying or hiding of the badge, the case for ternary operator instead of using `invisible={true}`.
In some cases, as shown in the screenshots above (chrome and safari), using `invisible` doesn't work.

Other than that, the README.md and README.old.md files!